### PR TITLE
Updated Information for University Resource Pages

### PR DESCRIPTION
**Pull request by elisavettorstensson**

### CODE REVIEW

**Comment by elisavettorstensson** [comment]
checked all the links and email addresses in the section 'Research Data Support, Guidelines, and Policies' and made changes to the links that didn’t work. In some cases, I checked for more up-to-date information and added a few extra links that I thought might be relevant to the content. I could not find a site associated with the University of Gothenburg's email address ([researchdata@gu.se](mailto:researchdata@gu.se)). I’m not sure if it’s a valid email address.

File: content/topics/university-rdm-resources.md [hunk 1]
```diff
@@ -147,22 +149,22 @@ The role of the data protection officer (dataskyddsombud) is to check that the G
     <tr>
       <td>Umeå University</td>
       <td><a href = "mailto:pulo@umu.se">pulo@umu.se</a></td>
-      <td><a href="https://www.aurora.umu.se/stod-och-service/rad-och-riktlinjer/juridik-och-personuppgifter/personuppgifter">Personuppgifter</a></td>
+      <td><a href="https://www.umu.se/en/about-the-website/legal-information/processing-of-personal-data/">Personuppgifter</a></td>
     </tr>
     <tr>
       <td>University of Gothenburg</td>
       <td><a href = "mailto:dataskydd@gu.se">dataskydd@gu.se</a></td>
```
Comment (elisavettorstensson) [inline content/topics/university-rdm-resources.md:156]: There is another email address: dataskyddsombud@gu.se. The email that we provide is for the dataskydd grupp but it works in some way. They offer these two alternatives as email contacts.

**Comment by elisavettorstensson** [comment]
I also checked the 'Data Protection Officers' section and made some changes. For some reason, I can’t view the preview locally (when using the preview in Visual Studio), so I’m not sure if the result of my work looks correct. I added the sites I found and updated some outdated information.

**Review by YvonneKallberg — APPROVED** [review]
I too have gone through all links, and updated were I see fit. The only thing missing is the updated version from Swedish Research Council, but as Elisavet says, it isn't available yet.

**Pull request merged**